src/cache-utils.ts

```diff
@@ -34,19 +34,19 @@ export const getPackageManagerInfo = async (packageManager: string) => {
 export const getCacheDirectoryPath = async (
   packageManagerInfo: PackageManagerInfo
 ) => {
-  let pathList = await Promise.all(
+  const pathList = await Promise.all(
     packageManagerInfo.cacheFolderCommandList.map(command =>
       getCommandOutput(command)
     )
   );
 
-  const emptyPaths = pathList.filter(item => !item);
+  const notEmptyPaths = pathList.filter(item => item);
```

Comment:
nitpick: It seems to me this variable name can be simplified to `paths` or `cachePaths`
```suggestion
  const cachePaths = pathList.filter(item => item);
```